src/core_plugins/kibana/public/dashboard/actions/metadata.ts

```diff
@@ -17,27 +17,33 @@
  * under the License.
  */
 
+import { Action } from 'redux';
 import { createAction } from 'redux-actions';
 
-export const deletePanel = createAction('DELETE_PANEL');
-export const updatePanel = createAction('UPDATE_PANEL');
-export const resetPanelTitle = createAction('RESET_PANEl_TITLE');
-export const setPanelTitle = createAction('SET_PANEl_TITLE',
-  /**
-   * @param title {string}
-   * @param panelIndex {string}
-   */
-  (title, panelIndex) => ({ title, panelIndex })
```

Comment:
This is the only interface change I made. I now expect the input to setPanelTitle to be the object rather than have this pointless little conversion step (which happened mainly bc I wasn't totally clear about how redux-actions worked when writing it the first time around).